### PR TITLE
devops: show package names instead of missing libs on Ubuntu 18.04

### PR DESCRIPTION
**Pull request by aslushnikov**
This patch starts putting package names to install on Ubuntu instead
of missing dependencies list.

The mapping of library to package name is obtained using the following
script: https://gist.github.com/aslushnikov/2766200430228c3700537292fccad064

References #2745

### CODE REVIEW

File: src/server/validateDependencies.ts [hunk 1]
```diff
@@ -34,15 +35,49 @@ export async function validateDependencies(browserPath: string, browser: Browser
...
       missingDeps.add(dep);
   }
   if (!missingDeps.size)
     return;
-  const deps = [...missingDeps].sort().map(dep => '   ' + dep).join('\n');
-  throw new Error('Host system is missing the following dependencies to run browser\n' + deps);
+  // Check Ubuntu version.
+  const missingPackages = new Set();
+  const osRelease = await readFileAsync('/etc/os-release', 'utf8').catch(e => '');
+  const [ubuntuName = '', ubuntuVersion = ''] = osRelease.trim().split('\n');
```
Comment (yury-s) [inline src/server/validateDependencies.ts:48]: Is the order of entries actually predefined? Not a big deal but using a map could be more reliable.

File: src/server/validateDependencies.ts [hunk 2]
```diff
@@ -34,15 +35,49 @@ export async function validateDependencies(browserPath: string, browser: Browser
...
+        missingDeps.delete(missingDep);
+      }
+    }
+  }
+
+  let missingPackagesMessage = '';
+  if (missingPackages.size) {
+    missingPackagesMessage = [
+      `  Install missing packages with:`,
+      `      apt-get install -y ${[...missingPackages].join('\\\n          ')}`,
```
Comment (yury-s) [inline src/server/validateDependencies.ts:64]: --no-install-recommends?

**Review by yury-s — APPROVED** [review]
Let's commit the script for collecting deps to this repo too?

File: src/server/validateDependencies.ts (hunk 1 above)
Comment (aslushnikov) [reply src/server/validateDependencies.ts]: fair enough. Done.

File: src/server/validateDependencies.ts (hunk 2 above)
Comment (aslushnikov) [reply src/server/validateDependencies.ts]: removed `-y` instead as per offline discussion

**Pull request merged**